tests/api/circulation/test_mails.py

```diff
@@ -84,6 +85,34 @@ def test_email_on_loan_checkout(
     assert msg.body == expected_body_plain
 
 
+def test_email_with_special_characters(
+    client, app_with_mail, users, testdata, loan_params
+):
+    """Test that special characters are correctly displayed."""
+    loan_data = testdata["loans"][1]
+    loan = Loan.get_record_by_pid(loan_data["pid"])
+    doc = Document.get_record_by_pid(loan_data["document_pid"])
+    doc["title"] = \
+        "Spèciâl chäráctèrs: " \
+        "`,~,!,@,#,$,%,^,&,*,(,),_,-,+,=,{,[,},},|,\\,:,;,',<,>,.,?,/,º,ª"
+    doc.commit()
+    db.session.commit()
+    with app_with_mail.extensions["mail"].record_messages() as outbox:
+        user_login(client, "admin", users)
+
+        assert len(outbox) == 0
+        current_circulation.circulation.trigger(
+            loan, **dict(loan_params, trigger="checkout")
+        )
+        assert len(outbox) == 1
+        msg = outbox[0]
+
+    expected_subject = (
+        """InvenioILS: your loan for "{0}" has started.""".format(doc["title"])
+    )
+    assert msg.subject == expected_subject
```

Comment:
Can you please check the body too? (pseudocode)
```suggestion
    assert msg.subject == expected_subject
    assert expected_subject in msg.body
```